analysis/analysis.py

```diff
@@ -370,6 +677,98 @@ def test_compute_total_view_time(self):
         total_view_time = compute_total_view_time(self.default_student_data)
         self.assertEqual(total_view_time, 0)
 
+    def test_avg_time_context(self):
+        """
+        Test the avg_time_context function to see if it can find the avg view times given a question
+        and context. Also tests for if the question or context isn't in the data set.
+        """
+        args = [self.test_student_data,
```

Comment:
Don't get into the habit of using this style for passing args -- it introduces an extra data structure and is unnecessarily hard to read. If you're trying to avoid a terribly long line the following is fine: 
```
avg_time = avg_time_context(self.test_student_data,
                            'In one word, how does this text make you feel?',
                            'This is an ad.')
```